### ISSUE TITLE
Official option code assignment for edns-client-subnet

### ISSUE THREAD

**Issue by fabled** [gdnsd/gdnsd#50]
The edns-client-subnet extension has been assigned official option code, and it needs support.

See:
https://groups.google.com/forum/#!topic/afasterinternet/HJD2WphubOg

The edns-client-subnet draft has been updated at:
http://datatracker.ietf.org/doc/draft-vandergaast-edns-client-subnet/

The related IANA registry listing this is at:
http://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-11

Basically, for transition, we need to support both operation codes, and when replying use the operation code that was in the query. After switch over date, the old operation code support can be removed.
